### DISCUSSION TITLE
Airgaped manager setup -- command failed with error 6

### DISCUSSION THREAD

**Discussion by brusef** [Security-Onion-Solutions/securityonion#13051] [Unsupported Versions] [answered]
Hello,
      This so-manager will not install.  When I deploy as an air-gapped manager in a distributed environment I get a failed installation.  the error code is "Command failed with error 6".  I'm not sure how to translate or where I can follow up to get relevant informaton.
Any thoughts?
![security-onion-error6](https://github.com/Security-Onion-Solutions/securityonion/assets/86001728/fd750f5c-fb42-465d-a31f-08b30068e749)

thanks!

**Comment by petiepooo** [comment]
it's hard to tell where the failure is with just that one line.  Can you please share the result of this command, run as root:  
```
grep -B10 "error 6" ~/sosetup.log
```

**Reply by brusef** [reply]
Thanks for that!  I was able to find this error.  It confirmed something I was concerned with.  
Even though I've set it up as "Airgap Distributed" it is still looking to call out to github.
Is it kosher that I comment that out on the the script?  How would I do that?

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/86001728/d4dbfe89-bd7d-40f5-98ab-064d6992bda6)

**Comment by dougburks** [comment]
Are you absolutely sure that you chose the Airgap option in this screen?
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/669d1a87-5ed8-4af8-83d7-2b9241a0e18e)

What is the output of the following command?
```
sudo grep "^Airgap:" /root/sosetup.log
```

Also, your screenshot shows that you're running 2.4.50 so you might want to try our 2.4.60 ISO image if possible.

**Reply by brusef** [reply]
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/86001728/7b4e881d-88bd-42ce-aef8-919d52dbb613)
 
yeah, i've been trying this for a couple of days and keep getting the same results.  I'll ask about using an updated ISO.

**Reply by dougburks** [reply]
How are you running Setup? 

The normal workflow is:
- install the ISO image
- reboot
- log in and Setup starts automatically
 
If you're running Setup that way, then choosing Airgap should avoid reaching out to Github. 

Are you running Setup some other way?

Also note that we released 2.4.70 on Wednesday:
https://blog.securityonion.net/2024/05/security-onion-2470-now-available.html

Regardless of which ISO image you use, make sure that you verify the ISO image as shown at:
https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/DOWNLOAD_AND_VERIFY_ISO.md

**Reply by brusef** [reply]
We were running so-setup ISO.  But it looks like there may have just been a problem with the version we were using or maybe the ISO.  We picked up a 2.4.50 ISO and it's doing great things.
Thanks so much for the feed back dougburks and petiepoo.

**Reply by dougburks** [reply, marked as answer]
Yes, you most likely had a corrupted ISO image. Please make sure that you always verify the the ISO image as shown at:
https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/DOWNLOAD_AND_VERIFY_ISO.md
